### PR TITLE
#5 add Swift Package Manager configuration

### PR DESCRIPTION
**Pull request by CAMOBAP**
- #5

### CODE REVIEW

**Comment by CAMOBAP** [comment]
Currently `Package.swift` up-to-date and valid.

### How it was tested

Screen Shot 2021-12-23 at 2 49 33 PM


I have tested with a dedicated local project, everything is works

<details>
  <summary>Sample SwiftUI Test View</summary>

```swift
//
//  HCaptchaTestView.swift
//  TestSPM
//
//  Created by CAMOBAP on 12/23/21.
//

import SwiftUI
import HCaptcha

struct PlaceholderView : UIViewRepresentable {
    var uiview = UIView()

    func makeUIView(context: Context) -> UIView {
        uiview.backgroundColor = UIColor.red
        return uiview
    }

    func updateUIView(_ view: UIView, context: Context) {
    }
}

struct HCaptchaTestView: View {
    private(set) var hcaptcha: HCaptcha!

    let placeholder = PlaceholderView()

    var body: some View {
        VStack{
            placeholder.frame(width: 400, height: 400, alignment: .center).foregroundColor(Color.red)
            Button(
                "validate",
                action: {
                    hcaptcha.validate(on: placeholder.uiview) { result in
                        print(result)
                    }
                }
            ).padding()
        }
    }


    init() {
        hcaptcha = try! HCaptcha()
        let hostView = self.placeholder.uiview
        hcaptcha.configureWebView { webview in
            webview.frame = hostView.bounds
        }
    }
}

struct HCaptchaTestView_Previews: PreviewProvider {
    static var previews: some View {
      HCaptchaTestView()
    }
}
```

</details>

### Issues

I have difficulties keeping two targets within one xcode project (one target is regular with pods, second one which with swift packages):

Screen Shot 2021-12-23 at 3 27 57 PM

### Result

I think it's impossible to keep both in the same project, and I feel like we don't need it because we do testing by `swift build` 

@e271828- could you please review and let me know what do you think

File: fastlane/Fastfile [hunk 1]
```diff
@@ -94,6 +95,19 @@ platform :ios do
     )
   end
 
+  desc "Build Swift Package"
+  lane :spm_build do
+    # swift build dones't support iOS build out of the box
```
Comment (e271828-) [inline fastlane/Fastfile:100]:
```suggestion
    # swift build doesn't support iOS build out of the box
```

**Comment by e271828-** [comment]
Looks good, please add `Sample SwiftUI Test View` code to README.

**Comment by CAMOBAP** [comment]
> Looks good, please add `Sample SwiftUI Test View` code to README.

Done

File: README.md [hunk 2]
```diff
@@ -147,6 +147,61 @@ let hcaptcha = try? HCaptcha(size: .compact)
 
 And you will now get the desired behavior.
 
+### SwiftUI Example
+
+Originaly `HCaptcha` designed to be used with UIKit. But you can easily use it with `SwiftUI` also
```
Comment (e271828-) [inline README.md:152]:
```suggestion
`HCaptcha` was originally designed to be used with UIKit. But you can easily use it with `SwiftUI` as well.
```

Comment (CAMOBAP) [reply README.md]: Done

**Review by e271828- — APPROVED** [review]

**Pull request merged**